dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java

```diff
@@ -249,6 +249,36 @@ private static List<String> getUserListFromWindows() throws IOException {
         return users;
     }
 
+
+
+    /**
+     * whether the user exists in linux
+     *
+     * @return boolean
+     */
+    public static boolean existTenantCodeInLinux(String tenantCode) {
+        try{
+            String result = exeCmd("id "+ tenantCode);
+            if (!StringUtils.isEmpty(result)){
+                return result.contains("uid=");
+            }
+        }catch (Exception e){
+            String message = e.getMessage();
+            //Because ShellExecutor method throws  exception to the linux return status is not 0
+            //not exist user return status is 1
+            if (message.contains("no such user")){
+                logger.warn(message);
```

Comment:
Is it better to use error instead of warn? Becasue the worker would not execute any task without user.